src/bytes.rs

```diff
@@ -79,19 +78,31 @@ use crate::{Symbol, SymbolOverflowError, DEFAULT_SYMBOL_TABLE_CAPACITY};
 enum Slice {
     /// True `'static` references.
     Static(&'static BStr),
```

Comment:
I think I'll move away from storing `BStr`s in the `Slice` and only convert to impl `Debug`. The reason to store the slices as `&BStr` is that it lets me be lazy and derive debug.